CHANGELOG.md

```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v3.1.0](https://github.com/voxpupuli/puppet-postfix/tree/v3.1.0) (2023-03-01)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-postfix/compare/v3.0.0...v3.1.0)
+
+**Implemented enhancements:**
+
+- Add option to select lookup table type [\#336](https://github.com/voxpupuli/puppet-postfix/pull/336) ([timdeluxe](https://github.com/timdeluxe))
+
+**Fixed bugs:**
+
+- Fix typos errors in postfix::satellite from PR 326 [\#333](https://github.com/voxpupuli/puppet-postfix/pull/333) ([cruelsmith](https://github.com/cruelsmith))
+- Handle \[host\] vs \[host\]:port nexthop [\#327](https://github.com/voxpupuli/puppet-postfix/pull/327) ([gcoxmoz](https://github.com/gcoxmoz))
+
+**Closed issues:**
+
+- Stop managing alias map \(/etc/aliases\) [\#337](https://github.com/voxpupuli/puppet-postfix/issues/337)
```

Comment:
Are these bugs or enhancements?

Reply:
I guess this should have been tagged skip-changelog… Updating…